### PR TITLE
move rest of tests to testament

### PR DESCRIPTION
**Pull request by ringabout**

### CODE REVIEW

**Comment by ringabout** [comment]
Why this fails? I can't understand. It shouldn't be OS specfic
```nim
2020-11-26T03:58:00.5999865Z tsums.nim(9)             tsums
2020-11-26T03:58:00.6000107Z assertions.nim(30)       failedAssertImpl
2020-11-26T03:58:00.6000373Z assertions.nim(23)       raiseAssert
2020-11-26T03:58:00.6000611Z fatal.nim(49)            sysFatal
2020-11-26T03:58:00.6001113Z Error: unhandled exception: tsums.nim(9, 8) `sumPairs(data) != 1.0`  [AssertionDefect]
2020-11-26T03:58:00.6001526Z 
2020-11-26T03:58:00.6001709Z FAIL: megatest execution failed
```

**Comment by timotheecour** [comment]
32 bit float => loss of precision

**Comment by ringabout** [comment]
@timotheecour 
Thanks for hints. You are really helpful!!!

File: tests/stdlib/tsums.nim [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import std/sums
+from math import pow
+
+var epsilon = 1.0
+while 1.0 + epsilon != 1.0:
+  epsilon /= 2.0
+let data = @[1.0, epsilon, -epsilon]
+assert sumKbn(data) == 1.0
+assert sumPairs(data) != 1.0 # known to fail
```
Comment (ringabout) [inline tests/stdlib/tsums.nim]:
```suggestion
# assert sumPairs(data) != 1.0 # known to fail in 64 bits
```

Comment (ringabout) [reply tests/stdlib/tsums.nim]: It is disabled now.

Or?
```nim
When sizeof(int) == 8:
  assert sumPairs(data) != 1.0 # known to fail
```

Comment (timotheecour) [reply tests/stdlib/tsums.nim]: Use when (likewise, when defined posix is used a lot in runnableExamples)

**Pull request merged**